packages/web3-utils/src/validation.ts

```diff
@@ -109,46 +110,36 @@ export const isTopicInBloom = isTopicInBloomValidator;
  */
 export const compareBlockNumbers = (blockA: Numbers, blockB: Numbers) => {
```

Comment:
What about using the 
```suggestion
export const compareBlockNumbers = (BlockNumberOrTag: Numbers, BlockNumberOrTag: Numbers) => {
```

Reply:
you mean: `compareBlockNumbers = (blockA: BlockNumberOrTag, blockB: BlockNumberOrTag)` and not `(BlockNumberOrTag: Numbers, BlockNumberOrTag: Numbers) ` , done